hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/ExecutionContext.java

```diff
@@ -61,6 +64,8 @@
     private final Set<Address> participants;
     private final Object executionLock = new Object();
     private final ILogger logger;
+    private final AtomicLong startTime = new AtomicLong();
```

Comment:
how is this different than the other start time above?

Reply:
You are comparing this agains `MasterJobContext.executionStartTime`, right? Then, assuming I'm not grossly misunderstanding something, the difference is that `ExecutionContext` has one instance for each cluster member so it's suitable for sourcing metrics (which are per cluster member) and that's what we do with the `startTime` from it. 

On the other hand `MasterJobContext.executionStartTime` exists only on the master node so we couldn't use it for the "execution start time" metric.